RELEASENOTES.docu

```diff
@@ -1724,4 +1724,11 @@ extern typedef struct { } my_type_t;</pre> </add-note></build-id>
   <build-id value="next"><add-note> Added an error if a device declares
       more than 63 log groups (including the two built-in log groups.)
   </add-note></build-id>
+  <build-id value="next"><add-note> Added a pair of warnings for if the
+      specified log level of a log statement is likely intended to instead
+      specify the log groups (<tt>WLGROUPSASLEV</tt>), or vice versa
+      (<tt>WLLEVASGROUPS</tt>) <bug number="22018374443"/>.
+      These warnings will only be enabled by default in Simics 7. In Simics 6
+      and below they must be explicitly enabled through the use of the
+      <tt>--warn</tt> flag. </add-note></build-id>
```

Comment:
We can consider merging the warnings to one? I doubt that anyone ever will only want one of them, and easier for the user to pass a single warning flag (and easier for us to explain it: "..through the use of the <tt>--warn=WLOGGROUP</tt> flag"). We can still parameterize the warning to give descriptive messages in both cases.

Reply:
Technically, it's enabled by default with SIMICS_API=7. But your wording is sufficient for a releasenote.

Reply:
True. How about `WLLEVGROUPSMIXUP` as the unified warning? Or maybe just `WLOGMIXUP`?